acme/acme/standalone.py

```diff
@@ -83,9 +83,17 @@ def __init__(self, ServerClass, server_address, *remaining_args, **kwargs):
                 new_address = (server_address[0],) + (port,) + server_address[2:]
                 new_args = (new_address,) + remaining_args
                 server = ServerClass(*new_args, **kwargs) # pylint: disable=star-args
-            except socket.error:
-                logger.debug("Failed to bind to %s:%s using %s", new_address[0],
+                logger.debug("Successfully bound to %s:%s using %s", new_address[0],
                     new_address[1], "IPv6" if ip_version else "IPv4")
+            except socket.error:
+                if self.servers:
+                    # Already bound using IPv6.
+                    logger.debug("Certbot wasn't able to bind to %s:%s using %s, this is often " +
+                    "expected due to the dual stack nature of IPv6 socket implementations.",
```

Comment:
Indentation is a bit off. 

Our developer guide has a section: [Coding Style](https://certbot.eff.org/docs/contributing.html#coding-style) which in turn points to [PEP-8](https://www.python.org/dev/peps/pep-0008/#indentation) and [Google Python Style Guide](https://google.github.io/styleguide/pyguide.html#Indentation), both of which describe preferred way to handle indentation. The links should take you directly to the correct part of the documents.